tracer/samples/IISInDocker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+### Stage: build
+FROM mcr.microsoft.com/dotnet/framework/sdk:4.8 AS build
+WORKDIR /app
+
+# copy csproj and restore as distinct layers
+COPY *.sln .
+COPY src/*.csproj ./src/
+COPY src/*.config ./src/
+RUN nuget restore
+
+# copy everything else and build app
+COPY src/. ./src/
+WORKDIR /app/src
+RUN msbuild /p:Configuration=Release
+
+### Stage: runtime
+FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2019 AS runtime
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+WORKDIR /inetpub/wwwroot
+COPY --from=build /app/src/. ./
+COPY startup.ps1 .
+
+ARG TRACER_VERSION=2.23.0
```

Comment:
We could do a curl on GH to get the latest release and parse the version as we do here and there

Reply:
Good idea! Done in 7802070e1201f7cd6dda492055acb4d00403eb9e